src/app/drag-and-drop/drag-and-drop.demo.html

```diff
@@ -0,0 +1,179 @@
+    </div>
+    <div class="row">
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 1</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 2</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 3</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="clr-example">
+    <h4>Basic Draggable</h4>
+    <div class="row">
+        <div class="col-xs-12">
+            <p>
+                Use <code class="clr-code">clrDragAndDropGroup</code> directive to make particular draggables interact
+                with particular droppables.
+            </p>
+            <button class="btn btn-primary" clrDraggable clrDragAndDropGroup="dnd-1">Draggable Button</button>
+            <button class="btn btn-success" clrDraggable clrDragAndDropGroup="dnd-2">Draggable Button</button>
+            <button class="btn btn-warning" clrDraggable clrDragAndDropGroup="dnd-3">Draggable Button</button>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-xs-4">
+            <div class="card" clrDroppable clrDragAndDropGroup="dnd-3">
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 1</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-xs-4">
+            <div class="card" clrDroppable clrDragAndDropGroup="dnd-2">
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 2</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-xs-4">
+            <div class="card" clrDroppable [clrDragAndDropGroup]="['dnd-1', 'dnd-2']">
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 3</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="clr-example">
+    <h4>Custom Ghosts</h4>
```

Comment:
Can we split this big file into different routes?

Reply:
Definitely, I will better organize and clean up the app demo part.